task_4/Task 1/index1.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Task 1</title>
+</head>
+<body>
+    <div style="border-style:solid;padding:2px; ">
+      <div style=" background-color:green;margin:20px;height:30px;line-height:30px;color: honeydew;">

```

Comment:
Will be better to use padding instead of height and line-height

Reply:
done